evaluate/eval.py

```diff
@@ -99,34 +104,35 @@ def eval_report(self, infer_labeled):
 
     def compute_total_repairs(self):
         query = "SELECT count(*) FROM " \
-                "(SELECT _vid_ " \
-                 "FROM %s as t1, %s as t2 " \
-                 "WHERE t1._tid_ = t2._tid_ " \
-                   "AND t1.attribute = t2.attribute " \
-                   "AND t1.init_value != t2.rv_value) AS t"\
-                %(AuxTables.cell_domain.name, AuxTables.inf_values_dom.name)
+                "  (SELECT _vid_ " \
+                "     FROM %s as t1, %s as t2 " \
+                "    WHERE t1._tid_ = t2._tid_ " \
+                "      AND t1.attribute = t2.attribute " \
+                "      AND t1.init_value != t2.rv_value) AS t" \
+                % (AuxTables.cell_domain.name, AuxTables.inf_values_dom.name)
```

Comment:
We should use `.format` with named interpolation parameters here.

I think we should also standardize on using `Template`s and replace all SQL query strings with them.